src/google/credential.rs

```diff
@@ -58,6 +68,17 @@ impl CredentialLoader {
 
     /// Load credential from pre-configured methods.
     pub fn load(&self) -> Result<Option<Credential>> {
```

Comment:
In case of the big breaking change, this should probably return the enum itself instead of only the service account variant